src/main.c

```diff
@@ -52,6 +52,7 @@
 #include <signal.h>
 #include <netdb.h>
 #include <setjmp.h>
+#include <getopt.h>
```

Comment:
The "getopt" fuction is located in the unistd.h header making this getopt.h include unnecessary (though it also contains the prototype).